src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+
+import * as config from "../../config";
+import {
+  BeneficialOwnerTypeChoice,
+  BeneficialOwnerTypeKey,
+} from "../../model/beneficial.owner.type.model";
+import { getBeneficialOwnerType } from "../../utils/beneficial.owner.type";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  getBeneficialOwnerType(req, res, next, false);
+};
+
+export const post = (req: Request, res: Response) => {
+  logger.debugRequest(req, `POST ${req.method} ${req.route.path}`);
+
+  return res.redirect(getNextPage(req.body[BeneficialOwnerTypeKey]));
+};
+
+// With validation in place we will only have 3 choices
```

Comment:
could use (beneficialOwnerTypeChoices as BeneficialOwnerTypeChoice) to assert you know more and remove undefined